src/libs/ajax/AzureStorage.js

```diff
@@ -69,7 +69,7 @@ export const AzureStorage = signal => ({
 
   listNotebooks: async workspaceId => {
     const notebooks = await AzureStorage(signal).listFiles(workspaceId, '.ipynb')
-    return _.map(notebook => ({ ...notebook, application: tools.Jupyter.label }), notebooks)
+    return _.map(notebook => ({ ...notebook, application: toolLabels.JupyterLab }), notebooks) //TODO: Tool label WAS Jupyter
```

Comment:
Why the TODO?

Reply:
To make sure I test it out, since the main change was Azure -> JupyterLab, not Jupyter -> JupyterLab